corehq/apps/app_manager/views/forms.py

```diff
@@ -122,23 +122,24 @@ def copy_form(request, domain, app_id, form_unique_id):
     app = get_app(domain, app_id)
     form = app.get_form(form_unique_id)
     module = form.get_module()
+    to_app = get_app(domain, request.POST['to_app_id']) if request.POST.get('to_app_id') else app
```

Comment:
Do we want to handle in case get_app fails?

Reply:
get_app handles if it fails; it raises a 404 if app_id is not found. 

(Which, to be honest, might be a little easy on the user, considering app_id comes from HQ, so if it's not found it's probably because the user has intentionally been messing with the POST values. Unfortunately computers don't offer the option of giving the user a strong negative user experience :smiling_imp: but if they invent keyboards or mice that can taze the user, please let me know. (Also curious why anyone would buy one of those.))

Reply:
will do :)